src/Core/Domain/Product/Pack/Query/GetPackedProductsDetails.php

```diff
@@ -28,24 +28,33 @@
 
 namespace PrestaShop\PrestaShop\Core\Domain\Product\Pack\Query;
 
+use PrestaShop\PrestaShop\Core\Domain\Language\ValueObject\LanguageId;
+use PrestaShop\PrestaShop\Core\Domain\Product\Pack\ValueObject\PackId;
 use PrestaShop\PrestaShop\Core\Domain\Product\ValueObject\ProductId;
 
 /**
  * Retrieves product from a pack
  */
-class GetPackedProducts
```

Comment:
Please define the BC Break

Reply:
I rename all the PackedProductDetails items into the original name